docs/PyOp.md

```diff
@@ -0,0 +1,66 @@
+# Python Operator 
```

Comment:
I would introduce this as Support for Language Interop and include the python operator as a sub-heading. Also, can you mark this as beta? like Python Operator (Beta).

Reply:
Indeed, logically we should orgnize them in the tree like:

InterOp
   |-------PyOP
   |-------CsOp
   |-------JavaOP

But would it be little bit overwhelming for clients who cares exclusively on either PyOp, CsOp or JavaOp?

Reply:
Ok, let's keep it as is.